node/runtime/src/lib.rs

```diff
@@ -85,7 +85,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
 	spec_version: 165,
```

Comment:
Shouldn't the spec version be increased (and not impl version)?

Reply:
@andresilva do I understand correctly that we increase the `spec_version` whenever we have breaking changes visible only outside of the runtime boundary?

This patch set does not touch the `core/authority-discovery/primitives::AuthorityDiscoveryApi`, thus as far as I can tell it is not visible outside of the runtime boundary. Am I missing something?

Reply:
Given that we switch to `Babe` as authority id, @andresilva is right. You change the behavior of runtime and this new runtime would not be compatible with the old runtime when it comes to authority discovery.

Reply:
My vague definition for when to increment `spec_version` is a bit stricter than that, since internal behavior of the module can change without any changes being introduced at the boundaries. Keeping the same `spec_version` means that a previous compiled blob of the runtime without these changes can be executed as is.